src/platform/test.rs

```diff
@@ -636,6 +638,48 @@ fn server_accept_first() {
                (data, vec![], vec![]));
 }
 
+#[test]
+fn server_try_accept_timeout() {
+    let (server, name) = OsIpcOneShotServer::new().unwrap();
+    let data: &[u8] = b"1234567";
+
+    thread::spawn(move || {
+        eprintln!("client: sleep");
+        thread::sleep(Duration::from_millis(1000));
+
+        eprintln!("client: connect");
+        let tx = OsIpcSender::connect(name).unwrap();
```

Comment:
Given the setup of this test, I would expect this to panic; maybe add an assertion that `connect` fails and join the thread on the main test thread? Background threads that will randomly* panic always feel kind of weird in tests

Reply:
Hmm, didn't see it panic. I'll check.

Reply:
Yea, it will probably mostly never visibly panic, due to the main thread exiting before it has a chance to panic in 99% of cases